test/index.js

```diff
@@ -27,6 +27,14 @@ test('throws when passed non-object', ({end, throws}) => {
   end()
 })
 
+test('returns original value when passed non-object [strict: false]', ({end, is, doesNotThrow}) => {
+  const redact = fastRedact({paths: ['a.b.c'], strict: false})
+  doesNotThrow(() => redact(1))
+  const o = redact(1)
+  is(o, 1)
+  end()
+})
```

Comment:
can you add a test with `strict: true` and one with the default?

Reply:
will do